documentation/1.0/content/setup/prerequisites.md

```diff
@@ -29,6 +29,8 @@ These include:
 - `openssl` - Used to generate an X.509 TLS certificate for ingress routes, should you ask the application
   to generate one for you.
 
+The WKT UI application is built using the Electron framework and as such, we support only the platforms and versions supported by [Electron](https://www.electronjs.org/docs/latest/tutorial/support#supported-platforms).  On Linux, we support only versions 8.0 and higher of Oracle Linux, RedHat Linux, and CentOS Linux.
```

Comment:
Please change "On Linux" to "For example"

Reply:
YES, good idea.

Reply:
Please change the last sentence to the following:

For example, we support only versions 8.0 and higher of Oracle Linux, RedHat Linux, and CentOS Linux, because of the Electron requirement for Fedora 24 or newer.